### DISCUSSION TITLE
Use an Async Select with Park-Ui

### DISCUSSION THREAD

**Discussion by BigDog1400** [cschroeter/park-ui#130] [Ideas]
Hi there, I am currently utilizing Park UI to build a component library for my project. However, I have encountered a requirement to create an async select component with a search feature that fetches data from an external resource. I am wondering if anyone has successfully implemented this functionality using Park UI, or if there is an alternative approach that can be taken.
I recall that in Chakra UI, the workaround involved passing styles from Chakra to the react-select/async component. I am curious if this method could be applicable to Park UI as well. Alternatively, I don't know if maybe there should be a feature request to add async select functionality to zagjs? and not depend on react-select
If anyone has any insights or experience with this issue, I would greatly appreciate your input. Thank you!

**Comment by cschroeter** [comment]
@BigDog1400 

I'm not 100% sure what you mean with an async select. I assume you want to load more items, the moment the user types sth right?

**Reply by BigDog1400** [reply]
Basically, yes, when the user type something, i want to trigger a function to fetch the items and then the returning options from that async function should displayed as elements in the select. Similar to this example https://codesandbox.io/p/sandbox/codesandboxer-example-2ynrmd
